sandbox/src/main/java/org/dcsa/conformance/sandbox/ConformanceWebuiHandler.java

```diff
@@ -448,38 +436,60 @@ private JsonNode _getAllSandboxes(String userId) {
         .getNonLockingMap()
         .getPartitionValuesBySortKey(
             "environment#" + accessChecker.getUserEnvironmentId(userId), "")
-        .values()
         .forEach(
-            sandboxNode ->
+            (key, value) ->
                 sortedSandboxesByLowercaseName.put(
-                    sandboxNode.get("name").asText().toLowerCase(), sandboxNode));
+                    value.get("name").asText().toLowerCase(),
+                    _loadSandbox(userId, key.substring("sandbox#".length()), false)));
     ArrayNode sandboxesNode = OBJECT_MAPPER.createArrayNode();
     sortedSandboxesByLowercaseName.values().forEach(sandboxesNode::add);
     return sandboxesNode;
   }
 
-  private JsonNode _getSandbox(String userId, JsonNode requestNode) {
-    String sandboxId = requestNode.get(SANDBOX_ID).asText();
+  private ObjectNode _loadSandbox(String userId, String sandboxId, boolean includeOperatorLog) {
```

Comment:
```suggestion
  private ObjectNode loadSandbox(String userId, String sandboxId, boolean includeOperatorLog) {
```
Please don't use the Python style naming convention anymore.